exercises/perfect-numbers/tests/perfect-numbers.rs

```diff
@@ -0,0 +1,30 @@
+extern crate perfect_numbers;
+
+use perfect_numbers::{Classification, classify};
+
+#[test]
+fn basic() {
+  assert_eq!(classify(0).unwrap_err(), "Number must be positive");
+}
+
+#[test]
+#[ignore]
+fn test_all() {
+  struct TestClassification {
```

Comment:
I'm interested in the fact that this file uses two-space indents whereas the example solution uses four-space indents. It is OK since the example solution won't be delivered to the student, so the two-space indents in this file are consistent, but was there a reason for it?

Reply:
this is weird, I reformatted the file to use the same standards as the other.